### DISCUSSION TITLE
custom mongod.conf file append to docker mongo it restarts

### DISCUSSION THREAD

**Discussion by emAdithyaShenoy** [docker-library/mongo#693] [General]
mongodb:

    container_name: mongodb

    hostname: mongodb

    image: mongo:4.4

    volumes:

      - "/etc/mongodb/data/db:/var/lib/mongodb/"

      - "/etc/mongodb/config/mongod.conf.orig:/etc/mongod.conf.orig"

    logging:

      driver: "json-file"

      options:

        max-size: "10m"

        max-file: "3"

    ports:

      - 27017:27017/tcp

    restart: unless-stopped

    command: ["mongod", "--config", "etc/mongod.conf.orig"]

i want to use the custom mongod.conf to append into the container for the settings

**Comment by whalelines** [comment]
This repository contains the source for the `mongo` Docker Official Image, https://hub.docker.com/_/mongo, and is not a general resource for running MongoDB.

You can ask for help in the [Docker Community Slack](https://dockercommunity.slack.com/). If you are not already a member of the Docker Community Slack, you can use [this link to join](http://dockr.ly/comm-slack). You can find more support options at [Docker Support](https://www.docker.com/support/), including documentation for [Docker Compose](https://docs.docker.com/compose/).

You may also want to check out [MongoDB support resources](https://www.mongodb.com/services/support).

When you do ask for help, I would recommend including more detail about what you are trying to do, what commands you are running, and what errors are being reported. Good luck!
